app/uk/gov/hmrc/transitmovementsconverter/models/ModelImplicits.scala

```diff
@@ -151,20 +155,21 @@ object ModelImplicits {
   implicit lazy val packagingType02Format                          = Json.format[PackagingType02]
   implicit lazy val packagingType03Format                          = Json.format[PackagingType03]
   implicit lazy val additionalSupplyChainActorTypeFormat           = Json.format[AdditionalSupplyChainActorType]
-  implicit lazy val previousDocumentType03Format                   = Json.format[PreviousDocumentType03]
-  implicit lazy val previousDocumentType07Format                   = Json.format[PreviousDocumentType07]
-  implicit lazy val previousDocumentType08Format                   = Json.format[PreviousDocumentType08]
-  implicit lazy val supportingDocumentType04Format                 = Json.format[SupportingDocumentType04]
-  implicit lazy val supportingDocumentType05Format                 = Json.format[SupportingDocumentType05]
-  implicit lazy val supportingDocumentType06Format                 = Json.format[SupportingDocumentType06]
-  implicit lazy val transportDocumentType01Format                  = Json.format[TransportDocumentType01]
-  implicit lazy val transportDocumentType02Format                  = Json.format[TransportDocumentType02]
-  implicit lazy val transportDocumentType04Format                  = Json.format[TransportDocumentType04]
+  implicit lazy val previousDocumentType03Format                   = formatWithTypeAdjusted(Json.format[PreviousDocumentType03])
```

Comment:
Ticket mentions PreviousDocumentType01, SupportingDocumentType01,
& TransportDocument01 - but we don't have them.